### DISCUSSION TITLE
Options limited to 50

### DISCUSSION THREAD

**Discussion by kkotha82** [orchidjs/tom-select#608] [Q&A] [answered]
I am generating the select box using dynamic information from the database, where a few hundred options are generated. However, tom select shows only 50. Is there any intrinsic limit to the number of options that can be shown? If yes, is there any way around this?

**Comment by kkotha82** [comment, marked as answer]
My apologies. Just noticed the "maxOptions" setting in the configuration. Problem solved 😀

**Reply by dmenisdev** [reply]
What I noticed is that you can have >50 selected items but the list to select from has a max of 50.

JSFiddle: https://jsfiddle.net/des8fk7c/

Try it out:
new TomSelect("#input-tags",{
	persist: false,
	createOnBlur: true,
	create: true
});

<div class="p-4"><input id="input-tags" value="awesome,neat, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60" autocomplete="off" placeholder="How cool is this?"></div>
